translations/hu/get5.phrases.txt

```diff
@@ -0,0 +1,387 @@
+    }
+    "MaxPausesTimeUsedInfoMessage"
+    {
+        "hu"            "A {2} felhasználta a maximális ({1}) időkérésre használható időt."
+    }
+    "MatchPausedByTeamMessage"
+    {
+        "hu"            "{1} taktikai időkérést kért."
+    }
+    "MatchTechPausedByTeamMessage"
+    {
+        "hu"            "{1} technikai időkérésért kért."
+    }
+    "PausesNotEnabled"
+    {
+        "hu"            "Az időkérések nincsenek engedélyezve."
+    }
+    "TechPausesNotEnabled"
+    {
+        "hu"            "A Technikai időkérések nincsenek engedélyezve."
+    }
+    "TechnicalPauseMidSentence"
+    {
+        "hu"            "technikai szünet"
+    }
+    "TacticalPauseMidSentence"
+    {
+        "hu"            "taktikai szünet"
+    }
+    "TimeRemainingBeforeAnyoneCanUnpausePrefix"
+    {
+        "hu"            "Ennyi idő maradt mielőtt bárki elindíthatja a mérkőzést"
+    }
+    "PauseTimeRemainingPrefix"
+    {
+        "hu"            "Hátralévő idő a szünetből"
+    }
+    "AwaitingUnpause"
+    {
+        "hu"            "Várakozás a folytatásra"
+    }
+    "PausedByAdministrator"
+    {
+        "hu"            "Egy admin leszünetelte a mérkőzést."
+    }
+    "PausedForBackup"
+    {
+        "hu"            "A játék vissza lett töltve a kör mentésből. Mindkét csapatnak be kell írnia az {GREEN}!unpause {NORMAL}parancsot a folytatáshoz."
+    }
+    "UserCannotUnpauseAdmin"
+    {
+        "hu"            "Egy admin hívta ezt az időkérést, ő is fogja tudni folytatni."
+    }
+    "PausingTeamCannotUnpauseUntilFreezeTime"
+    {
+        "hu"            "Az időkérést nem lehet visszavonni""
+    }
+    "PauseRunoutInfoMessage"
+    {
+        "hu"            "{1} csapatnak elfogyott a szüneteltetési ideje. Folytatódik a mérkőzés."
+    }
+    "TechPauseRunoutInfoMessage"
+    {
+        "hu"            "A technikai szünetre szánt idő elfogyott. Bárki tudja folytatni a mérkőést."
+    }
+    "TechPauseNoTimeRemaining"
+    {
+        "hu"            "A {1} csapatnak, nem maradt több ideje technikai időkérésre. Használja a taktikai időkérést."
+    }
+    "TechPauseNoPausesRemaining"
+    {
+        "hu"            "A {1} csapatnak, nem maradt több technikai időkérése. Használja a taktikai időkérést."
+    }
+    "TechPausePausesRemaining"
+    {
+        "hu"            "Megmaradt technikai időkérések a {1} csapatnak: {2}"
+    }
+    "MatchUnpauseInfoMessage"
+    {
+        "hu"            "{1} elindította a mérkőzést."
+    }
+    "WaitingForUnpauseInfoMessage"
+    {
+        "hu"            "{1} szeretné folytatni. Várakozunk a {2} csapatra, hogy beírják a {GREEN}!unpause {NORMAL}parancsot."
+    }
+    "PausesLeftInfoMessage"
+    {
+        "hu"            "Megmaradt taktikai időkérések a {1} csapatnak: {2}"
+    }
+	    "PrereleaseVersionWarning"
+    {
+        "hu"            "A Get5 ({1}) változatát futtatod ami kifejezetten fejlesztésre és tesztelésre van. Ez az üzenet kikapcsolható {2}."
```

Comment:
I think you need a "with" here: This message can be turned off "with" or "using" <the command>. Or is it right?

The English looks like "This message can be turned off with get5_disabled_update_notice."